index.js

```diff
@@ -175,6 +184,19 @@ alexa.response = function(session) {
   };
 };
 
+alexa.directives = function(directives) {
+  // load the alexa response directives information into details
+  this.details = directives;
+
+  this.set = function(directive) {
+    this.details.push(directive);
+  };
+
+  this.clear = function() {
```

Comment:
Thanks @dblock and @ericblade. I've also finished the dialogue code, but it depends on these changes for that to go. It will also need a decent chunk of specs, something I'll attempt to hobble together. If you know of any other collaborators or contributors that could help, please tag them here!